### ISSUE TITLE
a little difference in results.

### ISSUE THREAD

**Issue by JingsenZhang** [Tiiiger/bert_score#153]
Hi Tianyi!
With a constant random seed, my model is trained on the same training set and tested on the same test set, but the test results occasionally have a little difference (BERT-P value in the example, 4 decimal places) , 
Is this reasonable?
<img width="533" alt="截屏2023-03-20 21 36 57" src="https://user-images.githubusercontent.com/36886184/226356426-e77bba7e-a2d8-44ee-8faf-c7eeed9fdf71.png">
<img width="533" alt="截屏2023-03-20 21 36 52" src="https://user-images.githubusercontent.com/36886184/226356473-1b15e9ef-9c7f-431f-8f05-ac52feb18c73.png">
Thank you!

**Comment by JingsenZhang** [comment]
The values of bleu and rouge are unchanged: 
<img width="515" alt="截屏2023-03-20 21 40 57" src="https://user-images.githubusercontent.com/36886184/226357607-3ac4378a-4305-425f-b91a-4f8d4e1dd70d.png">

<img width="515" alt="截屏2023-03-20 21 40 48" src="https://user-images.githubusercontent.com/36886184/226357594-855fb57f-1840-427b-a00c-e5f1a5dd2bac.png">

**Comment by JingsenZhang** [comment]
doesn't matter

**Comment by YasamanJafari** [comment]
How did you fix this?
